.travis.yml

```diff
@@ -6,9 +6,9 @@ before_install:
     - "sh -e /etc/init.d/xvfb start"
 install:
     - "pip install -e git://github.com/edx/xblock-sdk.git#egg=xblock-sdk"
-    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements.txt"
-    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/test-requirements.txt"
+    - "make -C $VIRTUAL_ENV/src/xblock-sdk install"
     - "python setup.py develop"
-script: pep8 leaderboards --max-line-length=120 && pylint leaderboards && python run_tests.py --with-coverage --cover-package=leaderboards
+    - "mkdir var"  # for var/workbench.log
```

Comment:
We've put this in the `run_tests.py` script in other repos, which has the advantage that it works locally as well and not only on Travis, so I'd go with the same solution here.  In the end I think it's a bug in the workbench that it doesn't create the directory if necessary, but just creating the directory here is good enough.